### DISCUSSION TITLE
Compiling: Clarification in doc/compiling.md file

### DISCUSSION THREAD

**Discussion by AmitvikramPujar** [stlink-org/stlink#1317] [Q&A (resolved)] [answered]
Hi there ! I was building the `st-link` tools on `Ubuntu-23.04` system.
The instruction no. 6 in the [`st-link/doc/compiling.md`](https://github.com/stlink-org/stlink/blob/develop/doc/compiling.md#linux) file is:

> 6. Run `make package`to build a Debian Package. The generated packages can be found in the sub-directory `./build/dist`.

However, the generated packages are found in `./build/Release/dist` sub-directory. Just need clarification if the instruction no. 6 has a typographical error regarding the location. 

Thanks!

**Comment by Nightwalker-87** [comment]
Yes, this will be fixed soon. Thanks for the note.

**Comment by Nightwalker-87** [comment, marked as answer]
Fixed in commit 682ec389b720f164c42aafe1b0d7614abc29a12f.
